python-checks/src/main/java/org/sonar/python/checks/JwtVerificationCheck.java

```diff
@@ -45,26 +55,107 @@ public class JwtVerificationCheck extends PythonSubscriptionCheck {
     "python_jwt.verify_jwt",
     "jwt.verify_jwt"));
 
+  private static final Set<String> FQNS_WHERE_VERIFY_KWARG_SHOULD_BE_TRUE = new HashSet<>(Arrays.asList(
```

Comment:
better to move `FQNS` to the end and rename other contacts as well do get rid of mix of `FQNS` and `NAMES